### DISCUSSION TITLE
CloudFlare speedtest new tracking endpoint

### DISCUSSION THREAD

**Discussion by uBlock-user** [easylist/easylist#14519] [General]
@ryanbr `speed.cloudflare.com` now deploys one more additional tracking endpoint via `aim.cloudflare.com` post completion of speedtest. Please add `||aim.cloudflare.com^` to EP.

![Capture](https://user-images.githubusercontent.com/21290713/209907604-f79becc1-0f3b-4f05-9987-6f15ae8c9848.JPG)

**Comment by partingscientist** [comment]
@uBlock-user Fixed with https://github.com/easylist/easylist/commit/14b2430d50adca98a01aca560223b484f9430451

**Reply by uBlock-user** [reply]
Wow, took them almost a year to fix!
